Building on Debian 12: compile error

Hello, I was following the build guide, installed the recommended packages with apt, and libhpdf-dev (libhpdf-2.3.0).

However upon running make, I get compilation errors:

```
arf20@thinkpad:~/source/PrinterToPDF$ make -j16
gcc -O3 -funsafe-math-optimizations -I/usr/include/SDL -D_GNU_SOURCE=1 -D_REENTRANT -c PrinterConvert.c -o PrinterConvert.o
PrinterConvert.c: In function ‘_tiff_delta_printing’:
PrinterConvert.c:1190:71: error: ‘resolution360’ undeclared (first use in this function)
 1190 |       if (defaultUnit == 0) thisDefaultUnit = printerdpih * resolution360; // Default for command is 1/360 inch units
      |                                                             ^~~~~~~~~~~~~

PrinterConvert.c:1190:71: note: each undeclared identifier is reported only once for each function it appears in
make: *** [Makefile:59: PrinterConvert.o] Error 1
```

It appears resolution360 is never defined in the project:

```
arf20@thinkpad:~/source/PrinterToPDF$ grep -r "resolution360"
PrinterConvert.c:                if (defaultUnit == 0) thisDefaultUnit = printerdpih * resolution360; // Default for command is 1/360 inch units
PrinterConvert.c:                if (defaultUnit == 0) thisDefaultUnit = printerdpih * resolution360; // Default for command is 1/360 inch units
PrinterConvert.c:                if (defaultUnit == 0) thisDefaultUnit = printerdpiv * resolution360; // Default for command is 1/360 inch units
PrinterConvert.c:                if (defaultUnit == 0) thisDefaultUnit = printerdpiv * resolution360; // Default for command is 1/360 inch units        
```

--- a/PrinterConvert.c
+++ b/PrinterConvert.c
@@ -12,10 +12,10 @@
 #include "/usr/include/SDL/SDL.h"
 #include "dir.c"
 
-const char* version = "v1.10";
+const char* version = "v1.11";
 
 /* Conversion program to convert Epson ESC/P printer data to an Adobe PDF file on Linux.
- * v1.10
+ * v1.11
  *
  * v1.0 First Release - taken from the conversion software currently in development for the Retro-Printer module.
  * v1.1 Swithced to using libHaru library to create the PDF file for speed and potential future enhancements - see http://libharu.org/
@@ -44,6 +44,7 @@ const char* version = "v1.10";
  * v1.8 - Fixed some errors in graphics printing and double height text
  * v1.9 - Fixed potential errors with negative movements
  * v1.10- Fixed errors in the Delta Row Printing Mode
+ * v1.11- Added missing definition for resolution360
  *
  * www.retroprinter.com
  *
@@ -85,6 +86,8 @@ int cdpiv = 144;                            // fixed dots per inch used for prin
 int cpih = 10;                              // Default is PICA
 int cpiv = 6;                               // Default font height in cpi (default line spacing is 1/6" inch too)
 int dpih = 180, dpiv = 180;                 // resolution in dpi for ESC/P2 printers
+
+const float resolution360 = 1 / (float) 360; 
 
 // Space used for storage - printermemory holds the bitmap file generated from the captured data
 // seedrow is used for enhanced ESC/P2 printer modes where each line is based on preceding line

